source-code/website/src/pages/index/Header.tsx

```diff
@@ -1,189 +1,108 @@
-											clip-rule="evenodd"
-										/>
-									</svg>
-								</a>
-							</div>
-						</Show>
+										<link.Icon></link.Icon>
+										<span>{link.name}</span>
+									</a>
+								)}
+							</For>
+						</div>
 					</div>
-
-					{/* end icon social media   */}
-
-					<div class=" md:hidden z-50">
-						<Show
-							when={show() == false}
-							fallback={
-								<button
-									onClick={(e) => setShow(false)}
-									type="button"
-									class="inline-flex items-center justify-center  bg-background p-2 text-primary "
-									aria-expanded="false"
-								>
-									<span class="sr-only">Close menu</span>
-									{/* <!-- Heroicon name: outline/x-mark --> */}
-									<svg
-										class="h-6 w-6"
-										xmlns="http://www.w3.org/2000/svg"
-										fill="none"
-										viewBox="0 0 24 24"
-										stroke-width="1.5"
-										stroke="currentColor"
-										aria-hidden="true"
-									>
-										<path
-											stroke-linecap="round"
-											stroke-linejoin="round"
-											d="M6 18L18 6M6 6l12 12"
-										/>
-									</svg>
-								</button>
-							}
+					<div class="md:hidden z-50">
+						<button
+							onClick={() => setMobileMenuIsOpen(!mobileMenuIsOpen())}
+							type="button"
+							class="inline-flex items-center justify-center  bg-background p-2 text-primary "
 						>
-							<>
-								<button
-									onClick={(e) => setShow(true)}
-									type="button"
-									class="inline-flex items-center justify-center  bg-background p-2 text-primary "
-									aria-expanded="false"
-								>
-									<span class="sr-only">Open menu</span>
-									<svg
-										class="h-6 w-6"
-										xmlns="http://www.w3.org/2000/svg"
-										fill="none"
-										viewBox="0 0 24 24"
-										stroke-width="1.5"
-										stroke="currentColor"
-										aria-hidden="true"
-									>
-										<path
-											stroke-linecap="round"
-											stroke-linejoin="round"
-											d="M3.75 6.75h16.5M3.75 12h16.5m-16.5 5.25h16.5"
-										/>
-									</svg>
-								</button>
-							</>
-						</Show>
+							<span class="sr-only">
+								{mobileMenuIsOpen() ? "Close menu" : "Open menu"}
+							</span>
+							{mobileMenuIsOpen() ? (
+								<IconClose class="w-6 h-6"></IconClose>
+							) : (
+								<IconMenu class="w-6 h-6"></IconMenu>
+							)}
+						</button>
 					</div>
-
-					{/*Responsive icon end  */}
 				</div>
-				<Show when={show()}>
-					<div class=" absolute transform  shadow-md  transition border-t border-outline">
-						{/* <a href="/" class="flex items-center">
-								<img
-									class="h-8 w-auto "
-									src="/favicon/favicon.ico"
-									alt=" Company Logo"
-								/>
-								<span class=" self-center pl-2 text-xl font-bold">inlang</span>
-							</a> */}
-						<div class="   w-screen h-full bg-background ">
-							{/* Nav, responsive */}
-
-							<div class="p-4">
```

Comment:
if you have a single div with one class, chances are high that this div is unnecessary. In this case, the div was unnecessary. 

You could have just added p-4 to the parent div.